packages/worker/src/worker.ts

```diff
@@ -1116,6 +1116,7 @@ export class Worker {
                         cronScheduleToScheduleInterval: optionalTsToMs(cronScheduleToScheduleInterval) || undefined,
                         historyLength: activation.historyLength,
                         unsafe: {
+                          now: () => -1, // set in initRuntime
```

Comment:
Alternatively, I could make a new type that omits this field?

Reply:
Any reason not to set it to `Date.now` here instead of doing it in `initRuntime`?

Reply:
I didn't try, but it looked like this is outside the worker thread and a that function wouldn't make it through serialization https://nodejs.org/api/worker_threads.html#portpostmessagevalue-transferlist

Reply:
You don't need this here

Reply:
@bergundy Build fails without it. Do you prefer my type suggestion?

Reply:
I'd just make it `Date.now()` just in case anyone tries to access this in their sinks

Reply:
You can clear before transferring to the worker thread and recreate it there or in the workflow context.